### DISCUSSION TITLE
Daylight Saving Time question

### DISCUSSION THREAD

**Discussion by dieter-erich** [arendst/Tasmota#21124] [Support request]
Hi,
     In a Nous intelligent plug I set the timer 1 to ON at 7:00 and timer 2 to OFF at 7:01 but, apparently due to the change to summer time it now turns on at 8:00 and off at 8:01 despite continuing to show the set time as "7:00" to "7:01" in the GUI. This is confusing. Can anybody please tell me how to make it more consistent. I do not care about DST and it can well turn and and off an hour earlier in summer as long as it stays ON for one minute. The time change (or anything else that might have happened) seems to also have caused faulty switching: The plug stayed ON for a long time causing heavy flooding of my plants.....
thanks for hints

**Comment by barbudor** [comment]
See https://tasmota.github.io/docs/Commands/#timezone

Depending on how Tasmota is configured, either it follows an official timezone with DST, or you can force a fixed offset from UTC

Forced offset is what most users find confusing

**Reply by dieter-erich** [reply]
Thank you barbudor! Seems to be a complicated command.... Would it lead to displaying "7:00" as I entered for turning on and effectively executing the command at 7:00? My problem is simply that some inconsistency blocked turning OFF the switch. I do not care whether it happens at 7:00 or 8:00 it must only turn it OFF at 7:01 or 8:01, respectively......

**Reply by barbudor** [reply]
Without any logs provided and more details on your timer set up, it's hard to reach a conclusion.
Timer should be able to execute as specified.

If you are only interrested only by gett ON for 1 minute, I suggest you look into the configuration command PulseTime
After configuring a pulsetime of 1 minute, you just need a single timer event for ON and Tasmota will turnoff automatically after the programme pulsetime

**Reply by sfromis** [reply]
Using PulseTime is indeed a good plan for this.

When it comes to what happens, one risk factor is using those timers without checking the box to enable each timer, or not checking the box per weekday.

**Reply by dieter-erich** [reply]
Thanks for your help! Everything seems to work now and I do not care about the inconsistency between the shown ON time (7:00 - 7:01) and the real ON time (8:00 - 8:01). I guess that it remained ON must have been due to the changing time. I now read out the state (ON or OFF) every minute and can see that it works as it should. I also set some of the timers to OFF at 7:05, 7:10, 7:15, 7:20, and 7:30 to make sure that at least one of them turns it OFF.

**Reply by sfromis** [reply]
If you use the PulseTime approach, switching off will not depend on time of day, but simply that Tasmota will automatically turn off again after the interval.

I'd still recommend using the helper tool linked from the docs page to have settings without the confusion/inconsistency around what time of day it is.
